### PR TITLE
my-idea-abou-kivy.txt

### PR DESCRIPTION
**Pull request by aarmn80**
my idea about kivy

### CODE REVIEW

File: my-ideas [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<AARMN>
+
+1-I think it's better to add more widgets and layouts to it
+2-Kivy have a problem in right click that put red dots I wish in next version this ability be off as default
```
Comment (dessant) [inline my-ideas:4]: The red dots are multitouch emulation, they can be disabled with:
```python
from kivy.config import Config
Config.set('input', 'mouse', 'mouse,multitouch_on_demand')
```
or
```python
from kivy.config import Config
Config.set('input', 'mouse', 'mouse,disable_multitouch')
```

File: my-ideas [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+3-its designer project is unpleasent to use for RAD designers I wish for beta or stable versions
+4-incorrect position of touch in my linux device (you should touch upper than your widget to touch that widget
```
Comment (dessant) [inline my-ideas:6]: Please open a bug report at the Kivy repo, describing the issue in detail.
https://github.com/kivy/kivy

**Comment by dessant** [comment]
Thanks for your input, but this repository was maily used ([once](https://github.com/kivy/kivy-ideas/issues/1) 😋) for tracking issues not related to code.

**Comment by aarmn80** [comment]
Thanks alot

**Comment by aarmn80** [comment]
And excuse me 
I'm a new user of github and kivy to write bug report (that was no bug and I don't know how to use it) in ideas list

File: my-ideas (hunk 1 above)
Comment (aarmn80) [reply my-ideas:4]: What is difference between this two code

**Pull request closed without merging**